Rakefile

```diff
@@ -1,10 +1,8 @@
 require "yast/rake"
 
+Yast::Tasks.submit_to :casp10
```

Comment:
Use the 'casp10' target added in https://github.com/yast/yast-rake/pull/38 as it is already in CASP.